components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/model/UserRecoveryFlowData.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.recovery.model;
+
+import java.sql.Timestamp;
+
+/**
+ * This object represents an entry of the identity metadata database.
+ */
+public class UserRecoveryFlowData {
+    private String recoveryId;
+
+    private Timestamp timeCreated;
+
+    private int attempts;
+
+    private int resendCount;
+
+    public UserRecoveryFlowData(String recoveryId) {
+
+        this.recoveryId = recoveryId;
+    }
+    public UserRecoveryFlowData(String recoveryId, Timestamp timeCreated) {
+
+        this.recoveryId = recoveryId;
+        this.timeCreated = timeCreated;
+    }
+
+    public UserRecoveryFlowData(String recoveryId, Timestamp timeCreated, int attempts, int resendCount) {
+
+        this.recoveryId = recoveryId;
+        this.timeCreated = timeCreated;
+        this.attempts = attempts;
+        this.resendCount = resendCount;
+    }
+
+    public Timestamp getTimeCreated() {
+
+        return timeCreated;
+    }
+
+    public void setTimeCreated(Timestamp timeCreated) {
```

Comment:
Missing javadoc comment for some public methods in the class

Reply:
Fixed with 2cc465d0641dcf34e149842ef253d9157f6bf1d8